EPRN.Common.Data/DataModels/PrnHistory.cs

```diff
@@ -12,8 +13,8 @@ public class PrnHistory : IdBaseEntity
         public PrnStatus Status { get; set; }
 
         [MaxLength(200)]
-        [Required]
-        public string Reason { get; set; }
+        [AllowNull]
+        public string? Reason { get; set; }
```

Comment:
Don't think this is required, is it? Strings are nullable, and therefore just a regular string will denote a field that will allow nulls.

Reply:
Done.